### DISCUSSION TITLE
r128x-cli binary

### DISCUSSION THREAD

**Discussion by svavilov** [ShikiSuen/r128x#3] [General]
Hi! Could you include the r128x-cli compiled binary in the next release, please? Thanks!!!

**Comment by ShikiSuen** [comment]
@aviloff Oh. You want the `cli`. Okay. I'll make it available before October 2025.

**Comment by ShikiSuen** [comment]
@aviloff Instead of giving precompiled binaries, I am setting up homebrew tap. You may try:
```
brew tap shikisuen/r128x
brew install shikisuen/r128x/r128x
```

However, these setups are still under maintenance. The availability is still scheduled before October 2025.
